docs/release-notes/v11.1.0.md

```diff
@@ -0,0 +1,166 @@
+---
+id: v11.1.0
+title: Version v11.1.0 Release Notes
+sidebar_label: v11.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 25th, 2023_
+
+## Console
+
+### New Features
+
+#### User Groups Management
+
+From a dedicated section inside of the Company Overview, Company Owners can now manage Company users also through Groups.  
+Thanks to groups it is possible to simplify and speed-up the management of a multitude of users who have to be assigned the same privileges.  
+Management of users through groups sits alongside the actual individual management of users, to allow you to better structure access to resources as you prefer, giving you the freedom to choose how to govern access levels of your Company users.  
+
+![Groups section](./img/overview/group_table.png)
+
+To discover more about user groups management, just take a look at the related [documentation page](/development_suite/identity-and-access-management/group-management.md). 
+
+#### Privatized Marketplace for Companies
+
+From now on, Companies can have their own private Marketplace items, which will not be visible and usable from those who are not part of those specific Companies.   
+More specifically, Company Owners and Project Administrators can now be able to manage Marketplace resources that will be available exclusively for the members of their Company. 
+
+To discover more information on how to manage Marketplace resources owned exclusively by the Company, visit the dedicated Marketplace [documentation page](/marketplace/add_to_marketplace/contributing_overview.md#public-and-private-marketplace).
+
+### Improvements
+
+#### Check of Certificate Authority in cluster connection
+
+In Cluster creation, adding the Certificate Authority is actually not mandatory but it depends on the properties of each Cluster. It will always be checked: if the cluster needs a CA it will fail the connection if it has not been entered.
```

Comment:
```suggestion
During Cluster creation, adding the Certificate Authority is no more mandatory, based on your cluster setup you may still be required to provide a CA. Before connecting the Cluster to your Company, the Console will make sure that it can correctly reach the Cluster with a secure connection, meaning that if your Cluster uses self-signed certificate the CA will still be required.
```